blog/2020-10-15-how-temporal-simplified-checkr-workflows.md

```diff
@@ -25,13 +25,13 @@ release_version: V1.0.0
 _This case study is based on an interview with [Ben Jacobson](https://www.linkedin.com/in/bjacobso/) who was a key player in Temporal* adoption within Checkr_
 
 
-**Problem:**
+## Problem:
```

Comment:
Making text bold is not the same as making it a semantic heading. Docusaurus doesn't actually pick these up as headings by default.